pages/glossary/inclusive-descendant-element.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Inclusive descendant element
```

Comment:
Use https://dom.spec.whatwg.org/#concept-tree-inclusive-descendant instead

(visibility needs to be applied afterwards)